Gateway/Validator/CheckoutResponseValidator.php

```diff
@@ -89,6 +89,7 @@ public function validate(array $validationSubject)
                             }
                         }
                     } elseif (!empty($response['additionalData']['comprafacil.entity'])) {
+                        //Multibanco resultCode has changed after checkout v49 and comprafacil.entity is not received anymore
```

Comment:
it is now called entity without the prefix ? I see below that $multibancoData['entity']  is used.

Reply:
Yes, the index is now 'entity' with no prefix https://docs.adyen.com/payment-methods/multibanco/api-only#-payments-response